apps/studio/lib/constants/infrastructure.ts

```diff
@@ -118,11 +118,11 @@ export const PASSWORD_STRENGTH_COLOR = {
 }
 
 export const PASSWORD_STRENGTH_PERCENTAGE = {
-  0: '10%',
-  1: '30%',
-  2: '50%',
-  3: '80%',
-  4: '100%',
+  0: 10,
```

Comment:
Had to be transformed to number, `aria-valuenow` accepts only numbers. It was masked previously by `as any`.